.github/workflows/ci-unit-test.yml

```
@@ -16,7 +16,7 @@ jobs:
 
     - name: Install
       run: |
-        python3 -m pip install pytest mypy isort pyflakes
+        python3 -m pip install pytest mypy types-openssl-python isort pyflakes
```

With cryptography this is no longer necessary
```suggestion
        python3 -m pip install pytest mypy isort pyflakes
```